core/src/test/java/org/apache/iceberg/rest/RESTCatalogAdapter.java

```diff
@@ -344,7 +350,13 @@ public <T extends RESTResponse> T handleRequest(
           TableIdentifier ident = identFromPathVars(vars);
           return castResponse(responseType, CatalogHandlers.loadTable(catalog, ident));
         }
-
+      case REGISTER_TABLE:
+        {
+          Namespace namespace = namespaceFromPathVars(vars);
+          RegisterTableRequest request = castRequest(RegisterTableRequest.class, body);
+          return castResponse(
+              responseType, CatalogHandlers.registerTable(catalog, namespace, request));
+        }
```

Comment:
nit: add an empty line after the closing } to align with all the code inside this switch block